frontend/src/helpers/object.js

```diff
@@ -0,0 +1,3 @@
+export function objectDeepCopy(obj) {
+  return JSON.parse(JSON.stringify(obj));
```

Comment:
Hint: there is an alternative way of doing deep clone in a browser. More info can be found [here](https://developer.mozilla.org/en-US/docs/Glossary/Deep_copy).